app/controllers/organizations/adoptable_pets_controller.rb

```diff
@@ -3,7 +3,7 @@ class Organizations::AdoptablePetsController < Organizations::BaseController
 
   def index
     @pets = authorized_scope(
-      Pet.includes(:adopter_applications, images_attachments: :blob),
+      Pet.includes(:adopter_applications, :liked_pets, images_attachments: :blob),
```

Comment:
eager load a pet's liked pets for the index view. We need to tell a user if they have liked a pet or not, so for each pet card we will query this data to find out if their adopter_foster_account id is in the collection of the pet's liked_pets

Reply:
`    @likes = LikedPet.where(adopter_foster_account: current_user&.adopter_foster_account)
`